.github/workflows/python.yml

```diff
@@ -0,0 +1,43 @@
+name: Python
+
+on:
+  push:
+    branches:
+      - "master"
+  pull_request:
+    branches:
+      - "master"
+
+jobs:
+  lint_and_tests:
+    name: Lint and tests
+    env:
+      DJANGO_SETTINGS_MODULE: backend.settings.test
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
+      - uses: actions/cache@v2
+        id: cache
+        with:
+          path: ~/.cache/pip
+          key: ${{ runner.os }}-pip-${{ hashFiles('**/requirements.txt') }}
+          restore-keys: |
+            ${{ runner.os }}-pip-
+      - name: Install requirements
+        run: |
+          python -m pip install --upgrade pip
+          pip install --upgrade --upgrade-strategy eager -r backend/requirements.txt
+      - name: Install tools
+        run: pip install flake8 pep8-naming
+      - name: Lint
+        run: flake8 backend/
+      - name: Test
+        run: |
+          cd backend/
+          coverage run manage.py test
+          cd ..
```

Comment:
This doesn't seem necessary since there are no other actions:
```suggestion
```

Reply:
Yes, you are right.